src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -193,7 +193,7 @@ class RemoteServicesMockServer(port:Int) {
     mockServer.when(
       request()
         .withMethod("DELETE")
-        .withPath("/workflow/this_workflow_exists")
+        .withPath("/workflow/v1/this_workflow_exists")
```

Comment:
Aborts are performed with POSTS to an abort endpoint:

https://github.com/broadinstitute/cromwell#post-workflowsversionidabort